distribute.sh

```diff
@@ -116,7 +116,7 @@ fi
 
 case $OSTYPE in
   darwin*)
-    SED="sed -i ''"
+    SED="sed -i .orig"
```

Comment:
other approach which is nice is use gsed instead of sed.
That ensure coherent behavior

Reply:
good to know that it exists. Do you want me to use that one instead? probably it should work both ways I assume

Reply:
feel free to do as you prefer. I just wanted to mention you it existed as it can be very handy.
Many command line tools have their gnu version available on brew.